moxy-compiler/src/test/java/moxy/compiler/StrategyAliasTest.kt

```diff
@@ -92,6 +92,141 @@ class StrategyAliasTest : CompilerTest() {
+            @interface OneExecution {
+                @StateStrategyTypeTag
+                String tag() default "";
+            }
+            
+            public interface ViewInterface extends MvpView {
+                @OneExecution(tag = "testTag")
+                void testMethod();
+            }
+        """.toJavaFile()
+
+        val expected = generateViewInterfaceWithOneExecutionTestMethod("testTag")
+
+        val presenter = generateViewStateFor("moxy.ViewInterface")
+        val compilation = compileSourcesWithProcessor(viewInterface, presenter)
+
+        val expectedCompilation = compileSources(viewInterface, presenter, expected)
+
+        assertExpectedFilesGenerated(
+            compilation.generatedFiles(),
+            expectedCompilation.generatedFiles())
+    }
+
+    @Test
+    fun testAliasedStateStrategyWithoutTagSpecified() {
+        @Language("JAVA") val viewInterface = """
+            package moxy;
+            
+            import moxy.MvpView;
+            import moxy.viewstate.strategy.OneExecutionStateStrategy;
+            import moxy.viewstate.strategy.StateStrategyType;
+            import moxy.viewstate.strategy.StateStrategyTypeTag;
+            
+            @StateStrategyType(OneExecutionStateStrategy.class)
+            @interface OneExecution {
+                @StateStrategyTypeTag
+                String tag() default "";
+            }
+            
+            public interface ViewInterface extends MvpView {
+                @OneExecution
+                void testMethod();
+            }
+        """.toJavaFile()
+
+        val expected = GENERATED_VIEW_INTERFACE_WITH_ONE_EXECUTION_TEST_METHOD
+
+        val presenter = generateViewStateFor("moxy.ViewInterface")
+        val compilation = compileSourcesWithProcessor(viewInterface, presenter)
+
+        val expectedCompilation = compileSources(viewInterface, presenter, expected)
+
+        assertExpectedFilesGenerated(
+            compilation.generatedFiles(),
+            expectedCompilation.generatedFiles())
+    }
+
+
+    @Test
+    fun testWrongMultipleAliasTagMethods() {
+        @Language("JAVA") val viewInterface = """
+            package moxy;
+            
+            import moxy.MvpView;
+            import moxy.viewstate.strategy.OneExecutionStateStrategy;
+            import moxy.viewstate.strategy.StateStrategyType;
+            import moxy.viewstate.strategy.StateStrategyTypeTag;
+            
+            @StateStrategyType(OneExecutionStateStrategy.class)
+            @interface OneExecution {
+                @StateStrategyTypeTag
+                String tag() default "";
+                @StateStrategyTypeTag
+                String tag2() default "";
+            }
+            
+            public interface ViewInterface extends MvpView {
+                @OneExecution
+                void testMethod();
+            }
+        """.toJavaFile()
+
+        val presenter = generateViewStateFor("moxy.ViewInterface")
```

Comment:
Why viewState is called presenter here?

Reply:
That was a copy-paste.
This issue exists in all tests in this file.
Fixed all of them.